packages/gatsby/src/bootstrap/load-plugins/load.ts

```diff
@@ -71,9 +71,9 @@ export function resolvePlugin(
   rootDir = (!_.isString(plugin) && plugin.parentDir) || rootDir
 
   // Only find plugins when we're not given an absolute path
-  if (!existsSync(pluginName)) {
+  if (!existsSync(pluginName) && rootDir) {
```

Comment:
to appease TS as `rootDir` can be `undefined` - this need some verification if it will do what is expected tho before this PR is ready

Reply:
From my testing it seems correct, but will leave this unresolved for now